styles/picrew/catppuccin.user.css

```diff
@@ -477,15 +477,15 @@
     }
     .footer {
       background: none repeat scroll 0 @mantle !important;
-      border-top: 1px solid @surface2 !important;
+      border-top-color: @surface2 !important;
     }
     .small > a {
       color: @accent-color !important;
     }
     /* "Creator Informations" pages (creator/info page) */
     .card-header {
       border-color: @surface2 !important;
-      border-radius: 0px !important; /* necessary to hide a white background */
+      border-radius: 1px !important;
```

Comment:
The linter was't complaining about `0px` because you can't use zero, it was saying that you don't need the `px` unit because `0rem` is the same as `0em` which is `0px`.... and so on. So you can just do `0` instead of `0px`.

Anyway, can you explain why border-radius needs to be set here?

Reply:
When I tried theming that part, there would be a bit of unthemed white background behind where the corner would go, and I couldn't find any other fix for that at the time. I'll take another look at it and see if I can find a solution now

Reply:
No worries! Just wanted to make sure you knew why the linter was complaining :p

Reply:
So it turns out, I couldn't actually find the reason why I needed the border-radius at all :P It seems to be fine now, but I set background to @mantle just in case